Update (#68)
OpenVoiceOS/community-docs: docs/config_manual_config.md

# Configuration - Manually Change Files

> User configuration should be set in the `XDG_CONFIG_HOME` file.  Usually located at `~/.config/mycroft/mycroft.conf`.  This file may or may not exist by default.  If it does NOT exist, create it.

Reviewer: Consider a link to a site explaining what XDG is.

`mkdir -p ~/.config/mycroft`

`touch ~/.config/mycroft/mycroft.conf`

> Now you can edit that file.  To continue with the previous example, we will change the host of the TTS server, we will add the value manually to the Users `mycroft.conf` file.

Reviewer: ```suggestion
Now you can edit that file.  To continue with the previous example, we will change the host of the TTS server, then add the value manually to the user's `mycroft.conf` file.
```

Open the file for editing.  It is not uncommon for this file to exist, but be empty.

`nano ~/.config/mycroft/mycroft.conf`

> Enter the following into the file.  **NOTE** this file must be valid json or yaml format.  OVOS knows how to read both

Reviewer: ```suggestion
Enter the following into the file.  **NOTE** this file must be valid JSON or YAML format.  OVOS knows how to read both
```

Reviewer: Consider a link to a JSON linter (https://jsonlint.com/) and/or YAML linter (https://www.yamllint.com/)

```
{
  "tts": {
    "module": "ovos-tts-plugin-server",
    "ovos-tts-plugin-server": {
      "host": "https://pipertts.ziggyai.online"
    }
  }
}
```

> You can check the formating of your file with the `jq` command.

Reviewer: ```suggestion
You can check the formatting of your file with the `jq` command.
```

Reviewer: Consider mentioning most Linux distros don't ship with `jq` by default and providing links to installing it. I think most major distros have it available as `jq` so they can `apt install jq` or `yum install jq` or `pacman -u jq`

`cat ~/.config/mycroft/mycroft.conf | jq`

> If there are no errors, it will output the complete file.  On error, it will output the line where the error is.  You can use an online json checker if you want also.

Reviewer: ```suggestion
If there are no errors, it will output the complete file.  On error, it will output the line where the error is.  You can use an online JSON checker if you want also.
```
